.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,3 @@
+query-filters:
+  - exclude:
+      id: rb/csrf-protection-disabled
```

Comment:
What does Find a TRN do for this?

Reply:
Yeah, Find doesn't use this. We dismiss it whenever it pops up for the errors controller.

Reply:
I'll remove this, @tvararu has explained how this works in Find a TRN, I was getting a false positive on this rule because of linting.